client/src/main/scala/client/SimpleWebSocketClient.scala

```diff
@@ -1,40 +1,40 @@
 package client
 
-import monifu.concurrent.Scheduler
-import monifu.reactive._
-import monifu.reactive.channels.PublishChannel
+import monix.execution.{Ack, Cancelable, Scheduler}
+import monix.reactive.observers.Subscriber
+import monix.reactive.{Observable, Observer, OverflowStrategy, Pipe}
 import org.scalajs.dom.raw.MessageEvent
 import org.scalajs.dom.{CloseEvent, ErrorEvent, Event, WebSocket}
+
 import scala.concurrent.Future
 import scala.concurrent.duration._
 
 
-final class SimpleWebSocketClient private
-  (url: String, os: OverflowStrategy.Synchronous)
+final class SimpleWebSocketClient private(url: String, os: OverflowStrategy.Synchronous[String])
   extends Observable[String] { self =>
 
-  private def createChannel(webSocket: WebSocket)(implicit s: Scheduler) = {
+  private def createChannel(webSocket: WebSocket)(implicit s: Scheduler): Observable[String] = {
```

Comment:
Everything I said related to the back-pressured observable applies here as well.